packages/office-ui-fabric-react/src/components/Button/_base/Button.base.types.ts

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+import { ButtonBase } from './Button.base';
+import { IRenderFunction } from '../../../Utilities';
+import { IIconProps } from '../../../Icon';
+import { IStyleFunction } from '../../../Utilities';
+import {
+  IStyle,
+  ITheme
+} from '../../../Styling';
+
+export interface IButtonBase {
+  /**
+   * Sets focus to the button.
+   */
+  focus: () => void;
+}
+
+export interface IButtonBaseProps extends React.AllHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | ButtonBase> {
+  /**
+   * Optional callback to access the IButton interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IButtonBase) => void;
+
+  /**
+   * If provided, this component will be rendered as an anchor.
+   * @default ElementType.anchor
+   */
+  href?: string;
+
+  /**
+   * Changes the visual presentation of the button to be emphasized (if defined)
+   * @default false
+   */
+  primary?: boolean;
+
+  /**
+   * Unique id to identify the item. Typically a duplicate of key value.
+   */
+  uniqueId?: string | number;
+
+  /**
+   * Whether the button is disabled
+   */
+  disabled?: boolean;
+
+  /**
+   * Whether the button is checked
+   */
+  checked?: boolean;
+
+  /**
+   * Whether the button is expanded. Typically this is when a connected menu is open.
+   */
+  expanded?: boolean;
```

Comment:
Still feels weird to me to have this prop exposed even when the component can't render a menu